jobs/weather.rb

```diff
@@ -1,4 +1,5 @@
 
+require 'yaml'
```

Comment:
If launched separately, outside of dashing - failed with unrecongnized constants (YAML)